docs/upgrade-guides/core-2/nextjs.mdx

```diff
@@ -142,10 +142,16 @@ The most basic migration will be updating the import and changing out the defaul
 
 ```diff
 - import { authMiddleware } from "@clerk/nextjs/server"
-+ import { clerkMiddleware } from '@clerk/nextjs/server'
++ import { clerkMiddleware, createRouteMatcher } from '@clerk/nextjs/server'
```

Comment:
I can't review the sentence above, but its currently "The most basic migration will be updating the import and changing out the default export, then mirroring the previous behavior of protecting all routes as such:"

It should read move like "The most basic migration will be updating the import and changing out the default export, then mirroring the previous behavior of protecting all routes except /sign-in or /sign-up by doing the following:"